README.md

```diff
@@ -1,76 +1,110 @@
+[<img alt="ReDoc live demo" src="https://raw.githubusercontent.com/Redocly/redoc/master/demo/redoc-demo.png">](http://redocly.github.io/redoc/)
+
+
+
+<!-- [<img alt="Deploy to Github" src="http://i.imgur.com/YZmaqk3.png" height="60px">](https://github.com/Rebilly/generator-openapi-repo#generator-openapi-repo--) [<img alt="ReDoc as a service" src="http://i.imgur.com/edqdCv6.png" height="60px">](https://redoc.ly) [<img alt="Customization services" src="http://i.imgur.com/c4sUF7M.png" height="60px">](https://redoc.ly/#services) -->
 
 ## Features
-- Extremely easy deployment
-- [redoc-cli](https://github.com/Redocly/redoc/blob/master/cli/README.md) with ability to bundle your docs into **zero-dependency** HTML file
-- Server Side Rendering ready
-- The widest OpenAPI v2.0 features support (yes, it supports even `discriminator`) <br>
+
+All of these features are available in our open source community edition.
+
+- Deploy **extremely fast** via CDN, npm, CLI, or Docker.
+- Bundle your docs into a **zero-dependency** HTML file (when using [`redoc-cli`]()).
+- Integrate with any React app, with client or server-side rendering.
+- Take full advantage of OpenAPI 2.0/3.0 features, from `discriminator` to `allOf`.
+ <br> <br>
 ![](docs/images/discriminator-demo.gif)
-- OpenAPI 3.0 support
-- Neat **interactive** documentation for nested objects <br>
+- Use **interactive** documentation for nested objects
+ <br> <br>
 ![](docs/images/nested-demo.gif)
-- Code samples support (via vendor extension) <br>
+ <br> <br>
+- Include **code samples** (via vendor extension)
+ <br> <br>
 ![](docs/images/code-samples-demo.gif)
-- Responsive three-panel design with menu/scrolling synchronization
-- Integrate API Introduction into side menu - ReDoc takes advantage of markdown headings from OpenAPI description field. It pulls them into side menu and also supports deep linking.
-- High-level grouping in side-menu via [`x-tagGroups`](docs/redoc-vendor-extensions.md#x-tagGroups) vendor extension
-- Simple integration with `create-react-app` ([sample](https://github.com/APIs-guru/create-react-app-redoc))
-- Branding/customizations via [`theme` option](#redoc-options-object)
+ <br> <br>
+- Get a fully responsive, three-panel design out of the box (with menu/scrolling synchronization)
+- Integrate your API Introduction into the side menu (with support for deep linking)
+- Create high-level groupings in the side menu with the [`x-tagGroups`](docs/redoc-vendor-extensions.md#x-tagGroups) vendor extension
+- Match your docs and your brand by using the [`theme` option](#redoc-options-object)
+
+## Premium Features
+
+These features are availale in our hosted or on-premise premium edition.
+
+- SEO Friendly URLs
+- 10x Page Load Speed
+- 'Try It' API console
+  - Multiple Servers
+  - OAS3 security schemes
+- Enhanced Search
+- Additional Theming Control
+- Embedded Markdown
+
+## Live Examples
+- [Rebilly](https://rebilly-api.redoc.ly/)
```

Comment:
Should we separate premium examples from non-premium?  This link is also broken, check:

https://api-reference.rebilly.com/

Reply:
link fixed - in my opinion the paid/free distinction can be made when there are more than a dozen (and perhaps at redoc.ly instead of in the repo).